ocs_ci/utility/retry.py

```diff
@@ -24,6 +25,16 @@ def f_retry(*args, **kwargs):
                 try:
                     return f(*args, **kwargs)
                 except exception_to_check as e:
+                    if text_in_exception:
+                        if text_in_exception in str(e):
+                            logger.debug(
+                                f"Text: {text_in_exception} found in exception: {e}"
+                            )
+                        else:
+                            logger.error(
```

Comment:
Are we sure we want this at error level? This still feels like a debug level log entry to me.

Reply:
Right after this we will anyway fail with exception raised. I thought that this error log will help us to easier identify the reason.

Reply:
If we're raising the exception anyways then that seems like even more reason to not log this at error level. Since the user has specified a string they care about, I don't think we want to log at error level when that string wasn't matched.

That being said, the logic here looks good to me so I'm approving. We can always change the log level in a quick follow up PR.

Reply:
changed to debug level